recipes/gnu-time/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "time" %}
+{% set version = "1.7" %}
+{% set sha256 = "e37ea79a253bf85a85ada2f7c632c14e481a5fd262a362f6f4fd58e68601496d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://mirrors.ocf.berkeley.edu/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+
+test:
+  commands:
+    - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
+    - conda inspect objects -p $PREFIX $PKG_NAME   # [osx]
+    - $(which time) -v echo test
```

Comment:
Thoughts on using `command time` here? Not a big deal either way as this test works. Just a suggestion.